charts/vela/crds/standard.oam.dev_containerizeds.yaml

```diff
@@ -0,0 +1,5768 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
will this CRD generate automatically if we change the code?

Reply:
yes, the CRDs whose type are defined in our repo will change. The others are copied.

Reply:
cool